### ISSUE TITLE
Add site_edit capabilities to demag_gui

### ISSUE THREAD

**Issue by Swanson-Hysell** [PmagPy/PmagPy#27]
site_edit functionality can be added. What that program does is plot expected directions from two common errors - wrong mark on the sample and wrong way for the arrow. By doing so, samples can be excluded if there directions are consistent with such an error.

**Comment by Caoimhinmg** [comment]
Okay, This doesn't look too hard to patch over from the site_edit_magic.py script as it stands I think the major challenge will be the decision of how to implement. So on that note:
* As of now I'm thinking adding this to the interpretation editor is a good idea as the purpose is to edit interpretations and main GUI panel real estate is pretty thin. Also the interpretation editor already displays most of this info it just needs to plot wrong arrow, wrong mark, and wrong compass when requested then open dialogues to interact with the user.
* This does however complicate the plot which is already complicated for a user. So before working on this feature I think I should add in GUI context help which is built into wx and can be done with just a bit of work. (basically click on button or menu item then get question arrow which when clicked on some other part of the screen gives a help message popup box with info about that item) Once this is done it should simplify use and make it so that a new user isn't confounded by the complexity of this new function which promises to add 2 new symbols at least to the higher level mean plot at least which already has 3 possible symbols.
* To do this I'm going to add a file demag_gui_help.py which is basically just a script with all the messages in strings declared in variables so I can keep those out of the GUI and keep the code remotely readable. Any thoughts on where I should put something like this in the current directory structure? I was thinking help_files would work fine.

**Comment by Caoimhinmg** [comment]
Think this just needs debugging at this point, all of the features should be there with the last push. @Swanson-Hysell any suggestions for improvement or problems with the current form of this feature?

**Comment by Caoimhinmg** [comment]
As there have been no complaints about how this feature currently works I'm going to close this long standing issue. If there are any problems later just raise another issue and assign me.
